nautobot_ssot/integrations/infoblox/utils/client.py

```diff
@@ -1248,10 +1249,101 @@ def get_network_containers(self):
+
+        Returns:
+            (list) of record dicts
+
+        Return Response:
+        [
+            {
+                "_ref": "networkcontainer/ZG5zLm5ldHdvcmtfY29udGFpbmVyJDE5Mi4xNjguMi4wLzI0LzA:192.168.2.0/23/default",
+                "comment": "Campus LAN",
+                "extattrs": {},
+                "network": "192.168.2.0/24",
+                "network_view": "default",
+                "rir": "NONE",
+            },
+            {
+                "_ref": "networkcontainer/ZG5zLm5ldHdvcmtfY29udGFpbmVyJDE5Mi4xNjguMi4wLzI0LzA:192.168.2.0/23/default",
+                "comment": "Campus LAN 2",
+                "extattrs": {},
+                "network": "192.168.3.0/24",
+                "network_view": "default",
+                "rir": "NONE",
+            }
+        ]
+        """
+        url_path = "networkcontainer"
+        params = {
+            "_return_as_object": 1,
+            "_return_fields": "network,comment,network_view,extattrs,rir_organization,rir",
+            "_max_results": 100000,
+        }
+        params.update({"network_container": prefix})
+        response = self._request("GET", url_path, params=params)
+        response = response.json()
+        logger.info(response)
+        results = response.get("result", [])
+        for res in results:
+            res.update({"status": "container"})
+        return results
+
+    def get_child_subnets_from_container(self, prefix: str):
+        """Get child subnets from container.
+
+        Args:
+            prefix (str): Network prefix - '10.220.0.0/22'
+
+        Returns:
+            (list) of record dicts
+
+        Return Response:
+        [
+            {
+                "_ref": "network/ZG5zLm5ldHdvcmskMTAuMjIzLjAuMC8yMS8w:10.220.0.0/24/default",
+                "comment": "Campus 1",
+                "extattrs": {},
+                "network": "10.220.0.0/24",
+                "network_view": "default",
+                "rir": "NONE",
+                "vlans": [],
+            },
+            {
+                "_ref": "network/ZG5zLm5ldHdvcmskMTAuMjIzLjAuMC8yMS8w:10.220.1.0/24/default",
+                "comment": "Campus 2",
+                "extattrs": {},
+                "network": "10.220.1.0/24",
+                "network_view": "default",
+                "rir": "NONE",
+                "vlans": [],
+            },
+        ]
+        """
+        url_path = "network"
+        params = {
+            "_return_as_object": 1,
+            "_return_fields": "network,network_view,comment,extattrs,rir_organization,rir,vlans",
+            "_max_results": 10000,
+        }
+
+        params.update({"network_container": prefix})
+
+        try:
+            response = self._request("GET", url_path, params=params)
+        except HTTPError as err:
+            logger.info(err.response.text)
+            return []
+        logger.info(response.json())
+        return response.json().get("result")
```

Comment:
Keeping in sync with the approach from above

```suggestion
        response = response.json()
        logger.info(response)
        return response.get("result")
```